src/xcode/ENA/ENA/Source/Extensions/__tests__/UIFont+DynamicTypeTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import XCTest
+@testable import ENA
+
+final class UIFont_DynamicTypeTests: XCTestCase {
+    func testWeightFromString() {
+		XCTAssertEqual(UIFont.Weight("ultraLight").rawValue, UIFont.Weight.ultraLight.rawValue, accuracy: .high)
```

Comment:
Those tests would not be required if `@IBDesignable` + IB would not have been used in the first place. :)